NuGet.Services.Staging.sln

```diff
@@ -40,6 +38,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Client.Staging", "src
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Configuration", "src\NuGet.Services.Configuration\NuGet.Services.Configuration.xproj", "{E2D65F09-81A6-4D69-B128-C47C2DC43D36}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "NuGet.Services.Logging", "src\NuGet.Services.Logging\NuGet.Services.Logging.xproj", "{3A70A370-5ABE-4A53-B5B9-297DB288964E}"
+EndProject

```

Comment:
Nope, good catch. Classic "VS did not save my solution file" issue... :) Fixed in https://github.com/NuGet/NuGet.Staging/pull/72/commits/50333b13a8f3fd4579d72c0749d9a465eedbfd18